projects/ngx-intl-tel-input/src/lib/ngx-intl-tel-input.component.html

```diff
@@ -21,6 +21,7 @@
 					(keyup)="searchCountry()"
 					(click)="$event.stopPropagation()"
 					[placeholder]="searchCountryPlaceholder"
+					[autocomplete]="searchCountryAutocomplete"
```

Comment:
This parameter name doesn't match with the functionality. Please make it more readable. `searchCountryAutocomplete` maybe. And make it `boolean` not string.

Reply:
Thanks for the reply I will make it boolean. What do you mean with "`searchCountryAutocomplete` maybe", it's literally the same name i used.  Also do you think it should be enabled or disabled by default?

Reply:
Bro @webcat12345